src/System Application/App/ClientExtensions/src/BarcodeFailureReason.Enum.al

```diff
@@ -0,0 +1,22 @@
+namespace System.ClientExtensions;
```

Comment:
Copyright

// ------------------------------------------------------------------------------------------------
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See License.txt in the project root for license information.
// ------------------------------------------------------------------------------------------------